uint16                              tid
float32                             temperature
uint32                              sample_timestamp
uint32                              sync_timestamp
geometry_msgs/Accel                 accel
yesense_imu/YesenseImuQuaternion    quaternion
yesense_imu/YesenseImuEulerAngle    eulerAngle